.gitlab-ci.yml

```diff
@@ -3,5 +3,5 @@ image: docker:latest
 build:
   script:
       - docker login -u gitlab-ci-token -p $CI_JOB_TOKEN $CI_REGISTRY
-      - docker build --rm -t $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA .
+      - docker build --rm -t $CI_REGISTRY_IMAGE:$CI_COMMIT_SHA -f Docker/Dockerfile .
```

Comment:
It may be prefered to remove the --rm in the .gitlab-ci.yml (that way the cache will work).